exasol_script_languages_container_tool/lib/utils/language_def_parser.py

```diff
@@ -0,0 +1,66 @@
+from typing import List, Tuple, Union
+from urllib.parse import urlparse
+
+from exasol_script_languages_container_tool.lib.api.get_language_activation_builder import (
+    BuiltInLanguageDefinitionURL,
+    LanguageDefinitionURL,
+    SLCLanguage,
+)
+from exasol_script_languages_container_tool.lib.api.language_activation import (
+    SLCParameter,
+)
+
+
+def parse_language_definition(
+    lang_def: str,
+) -> Tuple[str, Union[LanguageDefinitionURL, BuiltInLanguageDefinitionURL]]:
+    alias_end = lang_def.find("=")
+    alias = lang_def[0:alias_end]
+    url = lang_def[alias_end + 1 :]
+    if url.startswith("builtin"):
+        lang = url.replace("builtin_", "")
+        for slc_builtin_language in SLCLanguage:
+            if slc_builtin_language.name.lower() == lang.lower():
+                return alias, BuiltInLanguageDefinitionURL(
+                    language=slc_builtin_language
+                )
+        raise ValueError(f"Unknown builtin language: {url}")
+    parsed_url = urlparse(url)
+    parameters = parsed_url.query.split("&")
+    language = ""
+    slc_parameters: List[SLCParameter] = list()
+    for param in parameters:
+        key, value = param.split("=")
+        if key == "lang":
+            language = value
+        else:
+            slc_parameters.append(SLCParameter(key=key, value=value))
+    # fragment is supposed to be something like:
+    # 'buckets/exaudf/exaudfclient_py3'
```

Comment:
Wrong it is either a relative path without buckets or an absolute with buckets. Both is actually legal. A udfclient doesn't need to be in the SLC container

Reply:
I changed the implementation for parsing the udf client (again). Pls check